src/Symfony/Component/HttpKernel/Tests/EventListener/RouterListenerTest.php

```diff
@@ -214,4 +218,61 @@ public function testRequestWithBadHost()
         $listener = new RouterListener($requestMatcher, $this->requestStack, new RequestContext());
         $listener->onKernelRequest($event);
     }
+
+    public function testResourceNotFoundException()
+    {
+        $this->expectException(NotFoundHttpException::class);
+        $this->expectExceptionMessage('No route found for "GET https://www.symfony.com/path" (from "https://www.google.com")');
+
+        $context = new RequestContext();
+
+        $urlMatcher = $this->getMockBuilder(\Symfony\Component\Routing\Matcher\UrlMatcherInterface::class)
+            ->disableOriginalConstructor()
+            ->getMock();
```

Comment:
```suggestion
        $urlMatcher = $this->createMock(\Symfony\Component\Routing\Matcher\UrlMatcherInterface::class);
```

plus, shouldn't we import the the class instead of writing the FQCN?

Reply:
I agree, but the whole test is like this. And I copied it. Should I fix the whole file?

Reply:
yes please

Reply:
Fixed.